content/20.pharmaceuticals.md

```diff
@@ -423,36 +432,29 @@ Other large-scale efforts to screen candidate drugs against SARS-CoV-2 (e.g., [@
-All four utilize existing bodies of knowledge in different ways.
-The first builds on clinical experience gained treating related diseases or related symptoms and allows for a rapid response aimed at saving the lives of patients already infected with a new disease.
-Oftentimes, the pharmaceutical agents that fall into this category are small-molecule, broad-spectrum pharmaceuticals that are widely available and affordable to produce.
-Clinically identified compounds are often available for other purposes, allowing clinicians to administer them to patients quickly either with an EUA or off-label.
+The range of approaches described here offer a range of advanctages and limitations in responding to a novel viral threat and build on existing bodies of knowledge in different ways.
+A host-oriented, symptom management approach allows clinicians to apply experience with related diseases or related symptoms to organize a rapid response aimed at saving the lives of patients already infected with a new disease.
+Oftentimes, the pharmaceutical agents that are applied are small-molecule, broad-spectrum pharmaceuticals that are widely available and affordable to produce, and they may already be available for other purposes, allowing clinicians to administer them to patients quickly either with an EUA or off-label.
 However, the fact that the targets of these agents are non-specific means that the mechanism of action can appear to be relevant to COVID-19 without a therapeutic or prophylactic effect being observed in clinical trials.
-Over the last year, dexamethasone has emerged as the strongest treatment against severe COVID-19, but HCQ/CQ and ARBs/ACEis have not borne out early hope in their mechanistic relevance.
-Although only limited testing of candidate therapies was possible during the SARS and MERS epidemics, the success of dexamethasone clearly illustrates how clinical context can be a valuable approach through which to identify candidate therapeutics.
-
-![
-**Mechanism of Action for Potential Therapeutics**
-Potential therapeutics currently being studied can target the SARS-CoV-2 virus or modify the host environment through many different mechanisms.
-Here, the relationship between the virus and several therapeutics described above are visualized.
-](images/N001-LifeCyclePlusDrugs.png){#fig:therapeutics secno=1}
-
-Two approaches seek to identify candidate pharmaceuticals based on information gained in non-clinical research on HCoV.
-The second approach reviewed above uses information about how related viruses function to identify possible ways to disrupt pathogenesis.
-These compounds are also typically small-molecule drugs, but often includes boutique and experimental medications like remdesivir,
+Over the last year, dexamethasone has emerged as the strongest treatment against severe COVID-19, but other clinically suggested compounds such as HCQ/CQ and ARBs/ACEis have not borne out early hope in their mechanistic relevance.
```

Comment:
Small molecule treatment? Would vaccines be included here?

Reply:
I thought the vaccines are biologics -- but I was only in the pharmacology department for eight months, so will definitely defer to you 😆  Regardless, we should probably mention vaccines in this discussion...
```suggestion
Over the last year, dexamethasone has emerged as the strongest treatment against severe COVID-19, but other clinically suggested compounds such as HCQ/CQ and ARBs/ACEis have not borne out early hope in their mechanistic relevance.
A number of vaccine development programs have also met huge success using a range of strategies [@individual-vaccines].
```

Reply:
It's up to you - I just wasn't sure if we were grouping or distinguishing between them. I'd consider this optional :)

Reply:
Thanks @cgreene! I'm curious what @RLordan thinks!

Reply:
I'm just as stumped haha, but if this helps the Center for Biologics Evaluation and Research regulates childhood vaccines in the US

Reply:
Thank you @RLordan and @cgreene!